dev/beaker/apply_manifests.rb

```diff
@@ -0,0 +1,16 @@
+test_name "Apply manifests for jenkins/git, sbt, and jjb"
+
+manifests = Dir.glob("./dev/manifests/*.pp")
+
+hosts.each do |host|
+  manifests.each do |local_path|
+    manifest_name = File.basename(local_path)
+    remote_path = File.join("/tmp/", manifest_name)
```

Comment:
I think maybe there's a method on the `Host` class in beaker where you can ask it for a temp file path.  That way you don't have to assume `/tmp/` exists, and it works with Windows, etc.  Not that I really care about Windows, but might as well use the idiomatic approach to doing this if possible.